relayer/client.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 // CreateClients creates clients for src on dst and dst on src if the client ids are unspecified.
-func (c *Chain) CreateClients(ctx context.Context, dst *Chain, allowUpdateAfterExpiry, allowUpdateAfterMisbehaviour, override bool, memo string) (bool, error) {
+func (c *Chain) CreateClients(ctx context.Context, dst *Chain, allowUpdateAfterExpiry, allowUpdateAfterMisbehaviour, override bool, customClientTrustingPeriod float64, memo string) (bool, error) {
```

Comment:
Additionally, I highly recommend changing the type here from `float64` to `time.Duration`. As written now, I don't know the unit of time (ms, seconds, minutes) from the name `customClientTrustingPeriod`.

Reply:
Done! time.duration is definitely the way to go. Thanks David.